ambari-server/src/main/java/org/apache/ambari/server/api/AmbariViewErrorHandlerProxy.java

```diff
@@ -51,14 +51,18 @@ public AmbariViewErrorHandlerProxy(ErrorHandler webAppErrorHandler, AmbariErrorH
 
   @Override
   public void handle(String target, Request baseRequest, HttpServletRequest request, HttpServletResponse response) throws IOException {
-
+   try{
     if (isInternalError(request, response)) {
       //invoke the ambari error handler
       ambariErrorHandler.handle(target, baseRequest, request, response);
     } else {
       //invoke the original errorhandler
       webAppErrorHandler.handle(target, baseRequest, request, response);
     }
+   }
+   catch(Exception e){
```

Comment:
Instead of generic exception, can we've specific exception.?  and both the handle methods throws IOE,so we can catch with same exception.